Cargo.toml

```diff
@@ -446,7 +446,7 @@ solana-last-restart-slot = "2.2.1"
 solana-lattice-hash = { path = "lattice-hash", version = "=2.2.16" }
 solana-ledger = { path = "ledger", version = "=2.2.16" }
 solana-loader-v2-interface = "2.2.1"
-solana-loader-v3-interface = "4.0.1"
+solana-loader-v3-interface = "5.0.0"
```

Comment:
this isn't ideal... there's a bunch of extra stuff that this bump is going to pick up. recall that 2.2 is stable now

Reply:
There is only one commit going from 4.0.1 to 5.0.0: https://github.com/anza-xyz/solana-sdk/commits/master/loader-v3-interface/src https://github.com/anza-xyz/solana-sdk/commits/master/loader-v3-interface/Cargo.toml

Reply:
ah apologies, i diffed the whole repo :melting_face: